inlang/source-code/sdk/src/createMessagesQuery.test.ts

```diff
@@ -19,6 +20,19 @@ describe("create", () => {
 		expect(created).toBe(true)
 	})
 
+	it("should create a message with a default alias", () => {
```

Comment:
The reasoning is missing in the test description on why a default alias should automatically be created. 

I don't understand why this should happen. This seems wrong due to implicit behaviour that we won't be able to remove in the future to ensure backwards compatibility. If this logic exists to ensure compatibility with `loadMessages()` and `saveMessags()`, I recommended to process every message passed to `saveMessages()` and (dynamically) create the dfault alias.

Reply:
Agree - changed the test description to 
`query.getByDefaultAlias should return a message with a default alias`